platforms/cpuinfo_test.go

```diff
@@ -0,0 +1,24 @@
+package platforms
+
+import (
+	"runtime"
+	"testing"
+)
+
+func TestCPUVariant(t *testing.T) {
+	if !isArmArch(runtime.GOARCH) || !isLinuxOS(runtime.GOOS) {
+		return
+	}
+
+	variants := []string{"v8", "v7", "v6", "v5", "v4", "v3"}
+
+	p := getCPUVariant()
+	for _, variant := range variants {
```

Comment:
I know there is still an [ongoing discussion](https://github.com/containerd/containerd/issues/1762) about testify/assert, but this is a case where using the assertion would make the test a lot more obvious:

```go
assert.Contains(t, variants, getCPUVariant())
```

This one line replaces 8 lines and provides better error messages.

Reply:
This code is type safe.